streamconfig/kafkaconfig/producer.go

```diff
@@ -91,7 +91,7 @@ type Producer struct {
 	// AckAll: broker will block until message is committed by all in sync
 	// replicas (ISRs).
 	//
-	// Defaults to `AckLeader`.
+	// Defaults to `AckAll`.
```

Comment:
Be aware this can have a speed impact since the default before was only 1 now it needs to wait to receive acks from all the replica's since we most of the time use 3 replicas per partition this means waiting on a ack from an extra host

Reply:
Yeah, that's true. But go-streamprocessor always had `AckAll` as the default, but now librdkafka does as well. So nothing changes in this case for us.